app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdViewEffectHandler.kt

```diff
@@ -0,0 +1,20 @@
+package org.simple.clinic.scanid
+
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedFactory
+import dagger.assisted.AssistedInject
+import org.simple.clinic.mobius.ViewEffectsHandler
+
+class ScanSimpleIdViewEffectHandler @AssistedInject constructor(
```

Comment:
I don't particularly see a point in making it part of the Dagger graph. Something that can be removed in the next PR?

Reply:
Oh yeah, you are right there. I initially made it this way so I could provide the schedulers, then realised later that it's not necessary and removed that. But I forgot to remove the Dagger graph related change.

Will make this change in another PR.